lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -93,15 +93,20 @@ class CommandInterpreterRunOptions {
   /// \param[in] add_to_history
   ///    If \b true add the commands to the command history. If \b false, don't
   ///    add them.
+  /// \param[in] process_repeats
+  ///    If \b true then process empty lines as repeat commands even if the
+  ///    interpreter is non-interactive.
   CommandInterpreterRunOptions(LazyBool stop_on_continue,
                                LazyBool stop_on_error, LazyBool stop_on_crash,
                                LazyBool echo_commands, LazyBool echo_comments,
                                LazyBool print_results, LazyBool print_errors,
-                               LazyBool add_to_history)
+                               LazyBool add_to_history,
+                               LazyBool process_repeats)
```

Comment:
nit: not sure what you mean by `process_repeats`, may be `command_repeats` would be make more sense here.

Reply:
I was trying to stay in line with the other parameter names in this function.  So for instance, add_to_history says commands should be added to the history.  The option I'm adding is to tell RunCommandInterpreter to process any repeat commands in the input stream.

Reply:
Internally in lldb we call the handling of newlines in the interactive interpreter as "repeat commands", and at the the CommandInterpreterRunOptions setting option was "AllowRepeats".

Reply:
> tell RunCommandInterpreter to process any repeat commands

Oh, `process` is a verb here --' when I read it I thought you were talking about the `Process` class which confused me a little ^^

Reply:
Sure.